gyb_syntax_support/Token.py

```diff
@@ -227,17 +227,6 @@ def macro_name(self):
     Punctuator('MultilineStringQuote', 'multiline_string_quote',
                text='\\\"\\\"\\\"', classification='StringLiteral'),
 
-    # Keywords prefixed with a '#'.
-
-    PoundKeyword('PoundKeyPath', 'keyPath', text='#keyPath'),
-    PoundKeyword('PoundLine', 'line', text='#line'),
-    PoundKeyword('PoundSelector', 'selector', text='#selector'),
-    PoundKeyword('PoundFile', 'file', text='#file'),
-    PoundKeyword('PoundFileID', 'fileID', text='#fileID'),
-    PoundKeyword('PoundFilePath', 'filePath', text='#filePath'),
-    PoundKeyword('PoundColumn', 'column', text='#column'),
-    PoundKeyword('PoundFunction', 'function', text='#function'),
-    PoundKeyword('PoundDsohandle', 'dsohandle', text='#dsohandle'),
     PoundKeyword('PoundAssert', 'assert', text='#assert'),
```

Comment:
`#assert` should be removable as well.

Reply:
We are still using it here

https://github.com/apple/swift-syntax/blob/3ca3c0bc15124009119d490e9aba998332f5dff5/Sources/SwiftParser/Statements.swift#L119-L120

Reply:
Alright, let's get rid of that later. It can be parsed as a freestanding macro expression.